tracking_manager/models/ir_model.py

```diff
@@ -73,8 +73,7 @@ def _get_model_tracked_by_o2m(self):
                 tracked_fields = custom_tracked_fields[model.model]
             else:
                 tracked_fields = model.field_id.filtered(
-                    lambda s: not s.readonly
-                    and not s.related
+                    lambda s: not s.related
```

Comment:
It will activate the tracking on all readonly fields which are related to the main model by a o2m field. 
No need code here, for exemple a sale order with lines (o2M field). You have to activate the custom tracking on the sale.order.line model and select the pertinent fields to track.

Reply:
I'll revert this change. I am not that deep into this part.